datagrid_gtk3/utils/transformations.py

```diff
@@ -105,6 +106,11 @@ def string_transform(value, max_length=None, oneline=True,
                 raise
             value = decode_fallback(value)
 
+    # Remove non-printable caracteres from the string. Only keep those
```

Comment:
Just spotted the same spelling mistake here as well (sorry I didn't point it out earlier)

`caracteres` -> `character`